pages/learn/develop/blocks.md

```diff
@@ -121,8 +121,9 @@ This would lead to your block being available on [balenaHub][blockhub] for folks
 
 ## Using your block in other projects
 
-The URL to pull your block image is available in balenaHub under the `Image Reference` field.
+The URL to pull your block image (the _image reference_) is available in balenaHub by clicking the `Use` button.
 
+<img src="/img/blocks/use-button.png" width="100%">
 <img src="/img/blocks/image-ref.png" width="100%">
```

Comment:
Use only the `Use` screenshot, since after clicking it you will see the image ref. No point keeping the latter image around. 
```suggestion
<img src="/img/blocks/use-button.png" width="100%">
```